test/__snapshots__/index.spec.ts.snap

```diff
@@ -828,6 +828,129 @@ import * as path from 'path';",
 ]
 `;
 
+exports[`stylistic rules > multiline-comment-style > fails on an invalid fixture 1`] = `
```

Comment:
forgot to include tests for stylistic plugin added recently, so fixing it in this PR